server/app/migrations/versions/04b11423b6b9_.py

```diff
@@ -0,0 +1,241 @@
+        sa.ForeignKeyConstraint(
+            ["spell_stat_id"],
+            ["spell_stats.uuid"],
+            name=op.f("fk_buff_spell_stat_id_spell_stats"),
+        ),
+        sa.PrimaryKeyConstraint("uuid", name=op.f("pk_buff")),
+        sa.UniqueConstraint("uuid", name=op.f("uq_buff_uuid")),
+    )
+    # ### end Alembic commands ###
+
+    dirname = os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    )
+
+    with open(os.path.join(dirname, "database/data/buffs.json"), "r") as file:
+        data = json.load(file)
+        session = Session(bind=op.get_bind())
+        conn = op.get_bind()
+
+        all_classes = data["spells"]
+        all_items = data["items"]
+
+        for d_class in all_classes:
+            for spell in all_classes[d_class]:
+                spell_name = spell["name"].replace("'", "''")
+                res = conn.execute(
+                    "select spell_id from spell_translation where spell_translation.name = '{}' and spell_translation.locale = 'en'".format(
+                        spell_name
+                    )
+                )
+                results = res.fetchall()
+
+                for level in spell["levels"]:
+                    res = conn.execute(
+                        "select uuid from spell_stats where spell_stats.spell_id = '{}' and spell_stats.level = '{}'".format(
+                            results[0][0], level["level"]
+                        )
+                    )
+                    level_results = res.fetchall()
+
+                    for buff in level["buffs"]:
+                        buff_object = ModelBuff(
+                            spell_stat_id=level_results[0][0],
+                            stat=to_stat_enum[buff["stat"]],
+                            min_value=buff["minValue"],
+                            increment_by=buff["incrementBy"],
+                            max_stacks=buff["maxStacks"],
+                            crit_min_value=buff["critMinValue"],
+                            crit_increment_by=buff["critIncrementBy"],
+                            crit_max_stacks=buff["critMaxStacks"],
+                        )
+                        session.add(buff_object)
+
+        session.commit()
+
+        for item in all_items:
+            res = conn.execute(
+                "select item_id from item_translation where item_translation.name = '{}' and item_translation.locale = 'en'".format(
+                    item["name"]
+                )
+            )
+            results = res.fetchall()
+
+            for buff in item["buffs"]:
+                buff_object = ModelBuff(
+                    item_id=results[0][0],
+                    stat=to_stat_enum[buff["stat"]],
+                    min_value=buff["minValue"],
+                    increment_by=buff["incrementBy"],
+                    max_stacks=buff["maxStacks"],
+                )
+                session.add(buff_object)
+
+        session.commit()
```

Comment:
I think we should move this to a script in `oneoff`. My reason for this is that if we keep this here, we may have problems in the future if we ever want to change the schema of your JSON in the future, and we'll always need to keep it backwards compatible with this migration.